test-apps/presentation-test-app/src/frontend/api/MyAppFrontend.ts

```diff
@@ -45,4 +55,35 @@ export class MyAppFrontend {
         label: spec.userLabel ?? spec.code.value!,
       }));
   }
+
+  public static async updateElement(imodel: IModelConnection, newProps: ElementProps) {
+    if (!IpcApp.isValid)
+      return;
+    return this.callIpc("updateElement", imodel.key, newProps);
+  }
+
+  public static async deleteElements(imodel: IModelConnection, elementIds: Id64Arg) {
+    if (!IpcApp.isValid)
+      return;
+    return this.callIpc("deleteElements", imodel.key, elementIds);
+  }
```

Comment:
IMO these two should throw when it's not an IPC app. That would make it very clear why they don't work.

Reply:
Changed